packages/perseus-core/src/utils/split-perseus-item.ts

```diff
@@ -5,6 +5,11 @@ import {getUpgradedWidgetOptions} from "../widgets/upgrade";
 
 import type {PerseusRenderer} from "../data-schema";
 
+/**
+ * get a copy of a Perseus item with pre-scoring data removed (ie answers)
```

Comment:
I think we agreed to call this rubric data. 
```suggestion
 * Return a copy of a Perseus item with rubric data removed (ie answers)
```